### PR TITLE
CB-171: Reset form when user clicks ‘Search’ - Composition

### PR DESCRIPTION
**Pull request by joel-ace**
# JIRA TICKET NAME:
[Reset form when user clicks ‘Search’ - Composition](https://issues.openmrs.org/browse/CB-171)

## SUMMARY:
This PR resets the composition form after it is submitted

### CODE REVIEW

File: tests/tabs/compositionComponent.test.jsx [hunk 1]
```diff
@@ -15,12 +15,6 @@ import sinon from 'sinon';
 import CompositionComponent from '../../app/js/components/tabs/tabcomponents/compositionComponent';
 
 describe('<CompositionComponent />', ()=>{
-    it('should mount the compositionComponent in the dom', ()=>{
```
Comment (andela-dbamidele) [inline tests/tabs/compositionComponent.test.jsx:18]: Is there a reason why you remove the test?

**Review by andela-dbamidele — COMMENTED** [review]
Awesome job!

**Review by mkiterian — APPROVED** [review]
Looks good to me

File: tests/tabs/compositionComponent.test.jsx (hunk 1 above)
Comment (joel-ace) [reply tests/tabs/compositionComponent.test.jsx:18]: Yes. It's not needed, as can be seen on line 30 on `compositionComponent.jsx`, componentDidMount was placed just so this test could be written. The other test passing shows that the component was actually mounted as the div elements won't be available if the component is not mounted

Comment (andela-dbamidele) [reply tests/tabs/compositionComponent.test.jsx:18]: Alright cool. 👍

**Pull request merged**